modules/update-network-sysctl-allowlist.adoc

```diff
@@ -0,0 +1,167 @@
+// Module included in the following assemblies:
+//
+// * nodes/containers/nodes-containers-sysctls.adoc
+
+:_content-type: PROCEDURE
+[id="updating-interface-specific-safe-sysctls-list_{context}"]
+= Updating the interface-specific safe sysctls list
+
+{product-title} includes a predefined list of safe interface-specific `sysctls`. Modify this list by updating the `cni-sysctl-allowlist` in the `openshift-multus` namespace.
```

Comment:
s/Modify/You can modify ?? Currently, it sounds to me that modifying the list is required.